### ISSUE TITLE
*#: in front of {{Unblock reviewed}} causes display glitch

### ISSUE THREAD

**Issue by NovemLinguae** [enterprisey/unblock-review#4]
Reported by jpgordon at https://en.wikipedia.org/wiki/User_talk:Enterprisey/unblock-review#slight_improvement

The following code

````
*:{{Unblock|reason=test}} ~~~~
````

works just fine, but once this script changes it to...

````
*:{{Unblock reviewed|decline=test|1=test}} ~~~~
````

it generates a formatting error. This can either be fixed in the {{Unblocked reviewed}} template, or we can have unblock-review delete asterisk, number sign, colon, and space between the newline and the template.

[Example diff.](https://en.wikipedia.org/w/index.php?title=User_talk:Samantha_J_20&diff=prev&oldid=1078696935&diffmode=source)

![image](https://user-images.githubusercontent.com/79697282/159590720-a97daad7-ab71-4459-a4cc-93f5b631fc7d.png)

**Comment by user-jpgordon** [comment]
_ we can have unblock-review delete asterisk, number sign, colon, and space between the newline and the template._ Wouldn't it be easier simply to insert a newline before the template?

**Comment by NovemLinguae** [comment]
True. Might leave stray bullets or big gaps though. The approach I was thinking of was a RegEx. Something like...

````
wikicode = wikicode.replace(/\n[*:# ]+{{Unblock/gi, '\n{{Unblock');

**Comment by user-jpgordon** [comment]
Makes sense;it's an easy enough regex, and the *:# have no purpose in life.

**Comment by user-jpgordon** [comment]
OK, it's been a while. Can we just have the newline? It's been happening more and more, as more people are using the "reply" tool, which shows up inside of block notices. I can't see it breaking anything that matters, and if it does, we'll fix it the same way we are already -- by making another edit.
